### ISSUE TITLE
Data to reproduce clip_coco.py

### ISSUE THREAD

**Issue by j-min** [eric-ai-lab/Mitigate-Gender-Bias-in-Image-Search#4]
Hi, thanks for sharing your code.
I'd like to use the clipping method you proposed. However, I couldn't find the `male_word_bank.txt`, `female_word_bank.txt`, `coco_precomp/testall_ids.txt`, `coco_testall_caps.txt`, `data/gender_coco.npy` files used in the `clip_coco.py`.
I also checked the https://github.com/mjskay/gender-in-image-search repo mentioned in README, but I couldn't find the data in the repo. Could you please guide and share the data needed to run `clip_coco.py`? Thanks!

**Comment by Faldict** [comment]
Hi, the `male_word_bank` and `female_word_bank` are the list of masculine and feminine words, respectively. You may personalize them. `testall_ids.txt` and `coco_testall_caps.txt` are available from the original COCO dataset. The `gender_coco.npy` is the annotated gender attirbutes.

**Comment by j-min** [comment]
I see, where can I get the `gender_coco.npy`?
Actually, could you please provide the [`mis`](https://github.com/eric-ai-lab/Mitigate-Gender-Bias-in-Image-Search/blob/main/clip_coco.py#L123-L129) indicating which dimensions of CLIP features have been clipped?
I'd like to experiment with if clipping the feature dimensions helps to reduce gender bias on another dataset.

**Comment by TobiasLee** [comment]
> I see, where can I get the `gender_coco.npy`? Actually, could you please provide the [`mis`](https://github.com/eric-ai-lab/Mitigate-Gender-Bias-in-Image-Search/blob/main/clip_coco.py#L123-L129) indicating which dimensions of CLIP features have been clipped? I'd like to experiment with if clipping the feature dimensions helps to reduce gender bias on another dataset.

Come to the same issue, and looking forward to the release of `gender_coco.npy` and the `mis`!

**Comment by itsyoavshalev** [comment]
Hi, can you please upload `male_word_bank`, `female_word_bank` and `gender_coco.npy` you used in your experiments?

Thanks.

**Comment by timqqt** [comment]
Hi, I believe the word banks are in the Section A of their Appendix. But are they all the ones used in the paper?

**Comment by Faldict** [comment]
> Hi, I believe the word banks are in the Section A of their Appendix. But are they all the ones used in the paper?

Yes, these words are the ones used for generating the gender attributes. However, you may need a keen eye on the unexpected words that may disclose the gender information.
